File: Programmers/Lv1_K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
	vector<int> answer;
	for (int i = 0; i < commands.size(); i++) {
		int begin = commands[i][0];
		int end = commands[i][1];
		int index = commands[i][2] - 1;
		vector<int> temp;
		for (int j = begin - 1; j <= end - 1; j++) {
			temp.push_back(array[j]);
		}
		sort(temp.begin(), temp.end());
		answer.push_back(temp[index]);
	}
	return answer;
}

File: Programmers/Lv1/Lv1_K번째수.py
def solution(array, commands):
    answer = []
    for i in commands:
        start, end, order = i
        answer.append(sorted(array[start-1:end])[order-1])
    return answer

array = [1, 5, 2, 6, 3, 7, 4]
commands = [[2, 5, 3], [4, 4, 1], [1, 7, 3]]
print(solution(array,commands))